Source/WebCore/html/FTPDirectoryDocument.cpp

```diff
@@ -266,7 +266,7 @@ void FTPDirectoryDocumentParser::parseAndAppendOneLine(const String& inputLine)
 
     String filename;
     if (result.type == FTPDirectoryEntry) {
-        filename = makeString(StringView { result.filename, result.filenameLength }, '/');
+        filename = makeString(std::span { result.filename, result.filenameLength }, '/');
```

Comment:
Should we extend `ListResult` with a `span` accessor? Or is this the only place we need to do this?